RIPS/rip-9995.md

```diff
@@ -142,7 +142,7 @@ In order for the transaction to be considered valid the
 sender validation frame MUST return two 64-bit values:
 
 ```
-ssz(uint64 validUntil, uint64 validAfter)
+ssz(bool success, uint64 validUntil, uint64 validAfter)
```

Comment:
bool is not compatible with adding an `authorizer` later.  That's why in 4337 it's not a bool - so that it can return an address.  Shouldn't we do the same here?